docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -81,10 +120,12 @@ tasks.named('compileKotlin', KotlinCompilationTask) {
 </tab>
 </tabs>
 
-Note that with the Gradle Kotlin DSL, you should get the task from the project's `tasks` first.
+Note that with the Kotlin Gradle DSL, you should get the task from the project's `tasks` first.
```

Comment:
I think it should stay as before—"Gradle Kotlin DSL" as it is already known term: https://docs.gradle.org/current/userguide/kotlin_dsl.html